data/Ousman37.json

```
@@ -0,0 +1,18 @@
+{
+  "name": "Ousman Ethmane",
+  "displayStatsPublic": true,
+  "bio": "Passionate Front-End Developer | UI/UX Enthusiast | Building innovative web experiences",
+  "avatar": "https://github.com/Ousman37.png",
+  "links": [
+    {
+      "name": "Follow me on LinkedI",
+      "url": "https://www.linkedin.com/in/ethmane-ousman-34885844/",
+      "icon": "FaLink"
+    },
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/Ousman37",
+      "icon": "FaGithub"
+    },
```

```suggestion
    }
```
The trailing comma isnot required